prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,21 @@
+var languages = [
+  'html',
+  'css',
+  'javascript',
+  'react',
+  'node',
+  'postgres'
+]
+;
```

Comment:
Not a huge deal but we typically want the closing bracket and the semicolon on the same line: `];`